tests/pd/test_rpc_client.rs

```diff
@@ -26,8 +26,7 @@ fn test_rpc_client() {
     };
 
     let client = RpcClient::new(&endpoints).unwrap();
```

Comment:
I guess it is easy to mock a gRPC PD server now, we can use this mock Server instead of real PD server. 

This can be considered later.